jarviscli/plugins/caesar_cipher.py

```diff
@@ -52,7 +52,7 @@ def plain_to_cipher(jarvis):
                 converted += i
         else:
             x = ord(i)
-            if x >= 192 and x <= 255:
+            if 192 <= x <= 255:
```

Comment:
Honestly didn't knew, this works in Python. But this is really nice! Thanks ;).

Reply:
Np!

Reply:
The way I think about it is if it works in English it should work in python.